src/build-commands/orb.yml

```
@@ -0,0 +1,68 @@
+version: 2.1
+
+description: |
+  A set of commonly used build commands for internal use.
+
+examples:
+  stoplight-build-commands-example:
+    description: "Checkout, set .npmrc, and clone down build scripts."
+
+    usage:
+      version: 2.1
+
+      orbs:
+        stoplight: stoplight/build-commands@0.0.1
+
+      jobs:
+        build:
+          docker:
+            - image: circleci/node:10-stretch
+          steps:
+            - checkout
+            - setup_remote_docker:
+                docker_layer_caching: true
+            - build-commands/clone_build_scripts
```

This should match whatever you name the orb on line 14: https://github.com/stoplightio/circleci-orbs/pull/1/files#diff-f51e35f24c88791a1367de6849f7b449R14

```suggestion
            - stoplight/clone_build_scripts
```